.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,25 @@
 ## FreeCodeCamp Pull Request template
-Please, go through steps before you submit a PR.
+Please, go through these steps before you submit a PR.
 
 1. Make sure that your PR is not a duplicate.
-2. If not then:
+2. If not, then make sure that:
 
-  2.1. Make sure you've done your changes in a separate branch, like `fix/signin-issue` or `feature/issue-templates`.
+  2.1. You have done your changes in a separate branch. Branches MUST have descriptive names that start with either the `fix/` or `feature/` prefixes. Good examples are: `fix/signin-issue` or `feature/issue-templates`.
 
-  2.2. You have a descriptive commit message.
+  2.2. You have a descriptive commit message. The first line of the commit message is the commit title - please remember to keep it short. Use more lines to provide more details.
 
   2.3. You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
 
-  2.4. Make sure `npm test` doesn't throw any error. If it does, fix them first and amend your commit.
+  2.4. `npm test` doesn't throw any error. If it does, fix them first and amend your commit (`git commit --amend`).
 
-3. **After** these steps you're ready to open a pull request. ***Don't*** forget to remove this template first.
+3. **After** these steps, you're ready to open a pull request. ***Don't*** forget to remove this template first.
 
   3.1. Give a descriptive title to your PR.
 
   3.2. Provide a description of your changes.
 
   3.3. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).
+  
+  3.4. Your pull request MUST target the `staging` branch on FreeCodeCamp's repository.
```

Comment:
Actually, this is high priority so should be 3.1 and not 3.4. If someone opens a PR against `master` branch, then it will be closed. It is not correct to say that PR MUST target the `staging`. It should say that no one should open a PR against `master` branch.